app/assets/stylesheets/components/_cards.scss

```diff
@@ -122,37 +122,37 @@
   }
 }
 
-// Smallest device
```

Comment:
Does masonry work on smaller devices without these?

Reply:
I think those are specifically for the grid you guys made, masonry should reshuffle the cards if the size changes
http://g.recordit.co/b3sOBHxsS9.gif